airflow/providers/vertica/hooks/vertica.py

```diff
@@ -31,11 +31,11 @@ class VerticaHook(DbApiHook):
     default_conn_name = 'vertica_default'
     supports_autocommit = True
 
-    def get_conn(self):
+    def get_conn(self) -> connect:
         """
-        Returns verticaql connection object
+        Return verticaql connection object
         """
-        conn = self.get_connection(self.vertica_conn_id)  # pylint: disable=no-member
+        conn = self.get_connection(self.vertica_conn_id)  # type: ignore # pylint: disable=no-member
```

Comment:
This is ignored by pylint and mypy because `VerticaHook.vertica_conn_id` overrides `DbApiHook.conn_name_attr`.

Is there a better way to handle this than ignoring?